### PR TITLE
Master salepocalypse missing tooltip aka

### PR DESCRIPTION
**Pull request by aka-odoo**
Task:
https://www.odoo.com/web#id=21157&view_type=form&model=project.task&menu_id=3942&action=327
Pad:
https://pad.odoo.com/p/r.wrfJa9wWLvSQZlhr

### CODE REVIEW

File: addons/purchase/purchase.py [hunk 1]
```diff
@@ -892,7 +892,9 @@ def _purchase_count(self):
     purchase_method = fields.Selection([
         ('purchase', 'On ordered quantities'),
         ('receive', 'On received quantities'),
-        ], string="Control Purchase Bills", default="receive")
+        ], string="Control Purchase Bills",
+        help="On ordered quantities: It will control purchase bills on quantity ordered.\n"
```
Comment (tivisse) [inline addons/purchase/purchase.py]: I can't see the added value on this tooltip.

File: addons/purchase/purchase.py [hunk 2]
```diff
@@ -892,7 +892,9 @@ def _purchase_count(self):
...
+        "On received quantities It will control purchase bills on quantity received.", default="receive")
```
Comment (tivisse) [inline addons/purchase/purchase.py]: Maybe the typos

File: addons/sale/sale.py [hunk 3]
```diff
@@ -824,7 +824,11 @@ def _sales_count(self):
 
 class ProductTemplate(models.Model):
     _inherit = 'product.template'
-    track_service = fields.Selection([('manual', 'Manually set quantities on order')], string='Track Service', default='manual')
+    track_service = fields.Selection([('manual', 'Manually set quantities on order')], string='Track Service',
+                                     help='Manually set quantities on order: It will create invoice for manually entered quanitity and no analytic account created.\n'
+                                     'Timesheets on contract: It will create invoice based on time tracking quantity (Like one hour, 5 hours).\n'
```
Comment (tivisse) [inline addons/sale/sale.py]: If I install only sale, I have a tooltip enumerating a selection which is actually not existing.

File: addons/sale/sale.py [hunk 4]
```diff
@@ -824,7 +824,11 @@ def _sales_count(self):
 
 class ProductTemplate(models.Model):
     _inherit = 'product.template'
-    track_service = fields.Selection([('manual', 'Manually set quantities on order')], string='Track Service', default='manual')
+    track_service = fields.Selection([('manual', 'Manually set quantities on order')], string='Track Service',
+                                     help='Manually set quantities on order: It will create invoice for manually entered quanitity and no analytic account created.\n'
```
Comment (tivisse) [inline addons/sale/sale.py]: Try to check yourself what you've written at least once, to avoid 'quanitity'

**Comment by dmo-odoo** [comment]
Merged at 83a35fa118e432de9aae28be363f56b554809bd0.

**Pull request closed without merging**